web_company_color/models/res_company.py

```diff
@@ -151,13 +151,24 @@ def button_compute_color(self):
             # Calc. optimal text color (b/w)
             # Grayscale human vision perception (Rec. 709 values)
             _a = 1 - (0.2126 * _r + 0.7152 * _g + 0.0722 * _b)
+            color_navbar_bg = n_rgb_to_hex(_r, _g, _b)
+            button_color_bg = n_rgb_to_hex(
+                *hls_to_rgb(_h, _l, _s + (0.25 if _s < 0.5 else -0.25))
+            )
+
             values.update(
                 {
-                    "color_navbar_bg": n_rgb_to_hex(_r, _g, _b),
+                    "color_navbar_bg": color_navbar_bg,
                     "color_navbar_bg_hover": n_rgb_to_hex(_rd, _gd, _bd),
                     "color_navbar_text": "#000" if _a < 0.5 else "#fff",
+                    "color_button_bg": button_color_bg,
+                    "color_button_bg_hover": "#fff" if _a < 0.5 else "#000",
```

Comment:
That's right.
After some tries I opted for a 20% increase in saturation.
For the moment I hardcoded a maximum _s value of 80% because I don't really know how we should handle high saturation colors.

Reply:
I'm open to other proposals as well